example/index.windows.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+import React, {
+  Component
+} from 'react';
+
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+import Video from 'react-native-video';
+
+class VideoPlayer extends Component {
+  constructor(props) {
+    super(props);
+    this.onLoad = this.onLoad.bind(this);
+    this.onProgress = this.onProgress.bind(this);
+  }
+
+  state = {
+    rate: 1,
+    volume: 1,
+    muted: false,
+    resizeMode: 'contain',
+    duration: 0.0,
+    currentTime: 0.0,
+  };
+
+  onLoad(data) {
+    this.setState({duration: data.duration});
+  }
+
+  onProgress(data) {
+    this.setState({currentTime: data.currentTime});
+  }
+
+  getCurrentTimePercentage() {
+    if (this.state.currentTime > 0) {
+      return parseFloat(this.state.currentTime) / parseFloat(this.state.duration);
+    } else {
+      return 0;
+    }
+  }
+
+  renderRateControl(rate) {
+    const isSelected = (this.state.rate == rate);
+
+    return (
+      <TouchableOpacity onPress={() => { this.setState({rate: rate}) }}>
+        <Text style={[styles.controlOption, {fontWeight: isSelected ? "bold" : "normal"}]}>
+          {rate}x
+        </Text>
+      </TouchableOpacity>
+    )
+  }
+
+  renderResizeModeControl(resizeMode) {
+    const isSelected = (this.state.resizeMode == resizeMode);
+
+    return (
+      <TouchableOpacity onPress={() => { this.setState({resizeMode: resizeMode}) }}>
+        <Text style={[styles.controlOption, {fontWeight: isSelected ? "bold" : "normal"}]}>
+          {resizeMode}
+        </Text>
+      </TouchableOpacity>
+    )
+  }
+
+  renderVolumeControl(volume) {
+    const isSelected = (this.state.volume == volume);
+
+    return (
+      <TouchableOpacity onPress={() => { this.setState({volume: volume}) }}>
+        <Text style={[styles.controlOption, {fontWeight: isSelected ? "bold" : "normal"}]}>
+          {volume * 100}%
+        </Text>
+      </TouchableOpacity>
+    )
+  }
+
+  render() {
+    const flexCompleted = this.getCurrentTimePercentage() * 100;
+    const flexRemaining = (1 - this.getCurrentTimePercentage()) * 100;
+
+    return (
+      <View style={styles.container}>
```

Comment:
>style [](start = 12, length = 5)

This was copied from the index.android.js, consider looking at index.ios.js to see if there's any other features we can easily add.